media/css/firefox/organizations.less

```diff
@@ -4,6 +4,9 @@
 
 @import "../sandstone/lib.less";
 
```

Comment:
This issue exists in prod currently, but in IE 10 (and maybe others?) the FAQ content bleeds off the edge of the page, making quite a bit of text unreachable/unreadable.

If you don't have time to address this here, we should file a bug. (Happy to file it if you're focused on getting this page out - just let me know.)

Reply:
Turns out I can test in IE8, 9, 11, and Edge but not 10.

I was able to replicate this in Firefox by falling back to the prefixed version so I suspect the problem was a conflict between the current spec and the 2012 syntax. I've attempted a quick fix for this by removing the prefixed flexbox declarations.